sec.club/src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,42 @@
+@import "../../variables.scss";
+
+.footer {
+    display: flex;
+    width: 100%;
+    border-top: $primary-color 1px solid;
+    z-index: 2000;
+    background-color: $background-color;
+    box-shadow: none;
+    flex-direction: row;
+    justify-content: space-evenly;
+    text-align: center;
+    align-items: center;
+
+    .icon-button {
+        margin: 0.8em;
```

Comment:
IMO this isn't needed and doesn't help appearance. If you want to add space between the border and the buttons, add `padding-top` to `.footer`